### ISSUE TITLE
loadtest and threadtest fails after recent update

### ISSUE THREAD

**Issue by sharkcz** [opencryptoki/openssl-ibmca#66]
I'm getting test failures in our CI after the last updates. The system is Fedora 33 on a KVM guest on a z14.
````
==============================================
   openssl-ibmca 2.1.2: test/test-suite.log
==============================================

# TOTAL: 23
# PASS:  21
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 2

.. contents:: :depth: 2

ERROR: loadtest
===============

Failed to create PKEY_CTX
Check for global variables failed!
ERROR loadtest (exit status: 99)

ERROR: threadtest
=================

Failed to create PKEY_CTX
Failed to create PKEY_CTX
Failed to create PKEY_CTX
Failed to create PKEY_CTX
Failed to create PKEY_CTX
Failed to create PKEY_CTX
Failed to create PKEY_CTX
Failed to create PKEY_CTX
Failed to create PKEY_CTX
Failed to create PKEY_CTX
Failed to create PKEY_CTX
Failed to create PKEY_CTX
Failed to create PKEY_CTX
Failed to create PKEY_CTX
Error in thread 0
Error in thread 1
Error in thread 2
Failed to create PKEY_CTX
Failed to create PKEY_CTX
Failed to create PKEY_CTX
Error in thread 3
Failed to create PKEY_CTX
Error in thread 4
Error in thread 5
Failed to create PKEY_CTX
Error in thread 6
Error in thread 7
Failed to create PKEY_CTX
Error in thread 8
Error in thread 9
Error in thread 10
Error in thread 11
Error in thread 12
Error in thread 13
Error in thread 14
Error in thread 15
Error in thread 16
Error in thread 17
Error in thread 18
Error in thread 19
ERROR threadtest (exit status: 99)
````

**Comment by sharkcz** [comment]
The previous build was on July 28 2020, so even the build environment changed since then. So it doesn't have to be strictly related to the last commits.

**Comment by p-steuer** [comment]
Thanks for reporting. loadtest and threadtest were added with the latest update ( a025995b7bcfe550209f502b27823b2a1911b7ec ) so it looks definitely related to the last update.

**Comment by juergenchrist** [comment]
Yes, sorry.  I only checked on z15.  Will check and fix for previous versions, too.

**Comment by sharkcz** [comment]
fixed in commit 281cb45a8514e616a777a7a5b93c1dd28ae428e9 in version 2.1.3
